forem/forem
# [15 min fix] ✂️  Remove unnecessary metadata for preview cards and sticky nav ✂️

## What type of PR is this? (check all applicable)


## Description

During the Profile Preview Card work we spotted that the code around organizations in `_user_meta_data` wasn't being used. Checking with product, there's no immediate plans to start showing this information in Profile Preview Cards or the article sticky nav (the only two places where `_user_metadata` is shown).

This PR removes that code around organizations, and brings the remaining content of the old `_user_metadata` file into `_profile_card_content` since it's the only place that uses it.

## Related Tickets & Documents

Follows on from discussions in #14056

## QA Instructions, Screenshots, Recordings

To test this, we just want to make sure there is no regression in Profile Preview Cards or in the sticky nav on the right-hand side of the article page. We should test this for both posts made by organizations and posts made by individual authors.

- Visit a post made by a user for an organisation
- Check the right-hand sticky nav shows the organisation name, tagline (if present), follow button, and CTA text/button (if present)
- Check that if you hover or click on the author name in the byline, the user metadata is present
- Visit a post made by a user _not_ for an organisation
- Check the right-hand sticky nav shows their user profile card
- Check that clicking/hovering on the author name in the byline shows the user metadata in the profile card
- Check that hovering over/clicking any comment author name shows the user's profile preview card, including their metadata


### UI accessibility concerns?

N/A - there shouldn't be any visible change as a result of this PR

## Added/updated tests?

- [X] No, and this is why: The existing tests continue to verify this functionality

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
- [X] This change does not need to be communicated, and this is why not: No change in behavior or UI

## [optional] Are there any post deployment tasks we need to perform?

N/A

## [optional] What gif best describes this PR or how it makes you feel?

![a hand holds a very tiny broom and gently sweeps up](https://media.giphy.com/media/2uIfee5OQRCmaAeqxJ/giphy.gif)
